website/src/components/ViewEventDrawer/ViewEventDrawer.test.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Drawer } from 'antd';
+import {
+  shallow,
+  configure,
+} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { ViewEventDrawer } from '..';
+
+configure({ adapter: new Adapter() });
+
+describe('Notification', () => {
+  /* enzyme testing */
+  const wrapper = shallow(<ViewEventDrawer visible onClose={() => {}} />);
+  it('Type of wrapper should be Drawer', () => {
+    expect(wrapper.type()).toBe(Drawer);
```

Comment:
You should also write snapshot tests. Maybe also test that the close functionality works.

Reply:
test is same as createeventdrawer

Reply:
Oh i did not see that can you update the tests for that too please. add at least snapshot tests.